package.json

```diff
@@ -204,6 +204,7 @@
     "@sentry/node": "^6.16.1",
     "@types/fast-levenshtein": "0.0.1",
     "@types/update-notifier": "^4.1.1",
+    "@yao-pkg/pkg": "^5.11.1",
```

Comment:
I see `pkg` has been deprecated. Would be good to know: why did we choose `@yao-pkg/pkg` over other forks?